src/content/docs/ai-monitoring/intro-to-ai-monitoring.mdx

```diff
@@ -0,0 +1,101 @@
+freshnessValidatedDate: never
+---
+
+import aiTraceViewIntroPage from 'images/ai_screenshot-full_trace-view-intro-page.webp'
+
+import aiAIResponsesOverview from 'images/ai_screenshot-full_AI-responses-overview.webp'
+
+When New Relic talks about AI, we're referring to the layer of your environment that uses a large language model (LLM) to generate a response when it receives an end user prompt. Your app contains an AI assistant, but behind the AI are libraries, cost implications, vector databases, and new processes that can be monitored.   
+
+<img
+    title="Trace waterfall for AI monitoring"
+    alt="A screenshot that shows the trace waterfall page for an individual AI response"
+    src={aiTraceViewIntroPage}
+/>
+
+<figcaption>
+    Go to **[one.newrelic.com](https://one.newrelic.com) > AI monitoring > AI responses**, then select a response row. 
+</figcaption>
+
+With AI monitoring, you can evaluate the performance of your AI by instrumenting your app with one of our APM agents. AI monitoring collects trace and span data about how your model generated a response, tracks performance metrics like latency and error rate, and gives you more oversight into your model's token usage. 
+
+## What is AI Monitoring? [#what-is-ai-monitoring]
+
+When your AI receives a prompt, an agent traces how the response generated through external LLMs, vector stores, and event data. From the UI, you can monitor performance trends through app metrics like uptime, latency, and error rate. With AI monitoring, you can:
+
+* **Evaluate errors and bugs in your code.** Use the responses table to identify AI responses that contain errors. Dig into the methods and calls your AI-powered app makes when generating a response. 
+* **Track token usage.** Did your prompt engineers make a recent change that affected cost? Use the AI response page to track tokens and keep costs down.
+* **Compare models.** If you're using different models in different environments, compare their performance against how much they cost.
+
+## Use AI monitoring to improve performance [#improve-performance]
+
+<img
+    title="AI responses page"
+    alt="A screenshot of the AI responses page"
+    src={aiAIResponsesOverview}
+/>
+
+<figcaption>
+    To overview your AI-powered app's performance: Go to **[one.newrelic.com](https://one.newrelic.com) > AI monitoring > AI responses**.
+</figcaption>
+
+Our AI responses page lets you overview how well your AI performs. See the number of responses generated in a given time period, then check how quickly your AI delivers responses. Are your end users waiting too long for a response? Did token usage spike recently? You can use the AI responses page to see general trends, then drill down by individual response, service, or model to improve your AI-powered app's performance. 
+
+## Get started with AI monitoring [#get-started-ai-monitoring]
```

Comment:
I like this doc overall but I feel like it needs a bit more to it, if possible. Basically I think an intro doc should answer the following questions:

What is this feature?
Who is this feature important for?
Where do I use this feature?
When and/or why would I want to use this feature?

I think this doc does a good job of answering the what and why, but if you could expand on the other's more it would help flesh the doc out and give users context before they continue, especially if you're going to move the pre-reqs to the install doc like we talked about this morning. The "where" specifically is a great place to use screenshots to give some meat to the doc.